keywords.txt

```diff
@@ -73,6 +76,9 @@ calibrate	KEYWORD2
 enable32K   KEYWORD2
 disable32K    KEYWORD2
 isEnabled32K    KEYWORD2
+enableClkOut    KEYWORD2
+disableClkOut    KEYWORD2
+isEnabledClkOut    KEYWORD2
```

Comment:
According to the Arduino [Library Specification][1], within keywords.txt

> the fields should be “separated by a single true tab (not spaces)”

Oh, wait, the three keywords just above your addition already have this problem... This will have to be fixed, maybe in another pull request.

[1]: https://arduino.github.io/arduino-cli/0.22/library-specification/#keywordstxt-format

Reply:
Good catch, it was a copy/paste then modify... but I should have noticed that the keyword was not recognized in the Arduino GUI! I think I will correct that in the next update.